Weapon Mods/Stable/OnyxArmamentCo/Data/Scripts/CoreParts/Heliod.cs

```diff
@@ -52,7 +52,7 @@ partial class Parts {
                 LockedSmartOnly = false, // Only fire    at smart projectiles that are locked on to parent grid.
                 MinimumDiameter = 0, // Minimum radius of threat to engage.
                 MaximumDiameter = 0, // Maximum radius of threat to engage; 0 = unlimited.
-                MaxTargetDistance = 0, // Maximum distance at which targets will be automatically shot at; 0 = unlimited.
+                MaxTargetDistance = 6000, // Maximum distance at which targets will be automatically shot at; 0 = unlimited.
```

Comment:
no this is cringe

Reply:
nevermind, it's actually based

Reply:
its actually an example of how to comment to lines